plugin/start-console.sh

```diff
@@ -2,6 +2,10 @@
 
 set -euo pipefail
 
+cd ..
+make .ensure-oc-login
+
+cd plugin
```

Comment:
an alternative to the multiple "cd" commands to move around, you can just use the `-C` / `--directory` option to `make`:
```suggestion
make --directory .. .ensure-oc-login
```

Reply:
Or, if you don't want to require "make" in the path, you can just do the same check the makefile does. something like:

```
if ! oc whoami &> /dev/null; then
  echo "You are not logged into an OpenShift cluster. Run 'oc login' before continuing."
  exit 1
fi
```